### PR TITLE
Only prune our own images

### PR DESCRIPTION
**Pull request by Mart124**
Hi,

This PR makes sure pruning is done on Bitwarden images only, now that images have a label (#305).
This solves [this community bug report](https://community.bitwarden.com/t/installing-bitwarden-into-an-existing-docker-server/1172).
Perhaps we could add `-a` to `docker image prune` to recover disk space. Perhaps this was the intended behavior ?

Thank you 👍

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -130,7 +130,8 @@ function dockerComposePull() {
 }
 
 function dockerPrune() {
-    docker image prune -f
+    docker image prune -f --filter="label=com.bitwarden.product=bitwarden"
+    # Perhaps we could add "-a", to recover disk space after automatic update ?
```
Comment (kspearrin) [inline scripts/run.sh]: no need for comments here, this could be part of the discussion in this PR comments

**Comment by Mart124** [comment]
Thank you for your review @kspearrin, done.
Of course feel free to squash !

**Comment by kspearrin** [comment]
I'm not sure if we want to use `-a`. Will that remove the setup container too since it wont be running at the time?

**Comment by Mart124** [comment]
Yes, there is chance.
And as we need it for the `updateDatabase` operation...
Perhaps we could move `dockerPrune` in the `update` section, right after `updateDatabase` ?
Not sure we need to prune every time we restart ?

**Comment by Mart124** [comment]
Thx 👍

**Pull request merged**